applications/flash_attention_v2/kernel/xe_flash_attn_decode.hpp

```diff
@@ -296,6 +298,28 @@ class FMHADecode {
       auto pVgV = thr_prefetch_V.partition_S(gV_prefetch);
 
       int kv_tile_idx = sub_group_id / ATOM_N;
+      int kv_cache_tile_idx = kv_tile_idx;
+      int tiles_per_page = ceil_div(mainloop_params.page_size, QK_SG_N);
+
+      if constexpr (PagedKV) {
+        if (seq_len_kv_cache != 0) {
+          // get physical page idx from page table
+          int curr_batch_pages = is_var_len ? mainloop_params.num_pages_per_seq[batch_coord + 1] - mainloop_params.num_pages_per_seq[batch_coord]
+                                            : ceil_div(seq_len_kv_cache, mainloop_params.page_size);
+          int curr_page_logical_idx = (kv_tile_idx * QK_SG_N) / mainloop_params.page_size;
```

Comment:
```suggestion
          int curr_page_logical_idx = kv_tile_idx * QK_SG_N / mainloop_params.page_size;
```
redundant parenthesis